### ISSUE TITLE
NPE when bootleg pod is required on latest bb

### ISSUE THREAD

**Issue by retrogradeorbit** [retrogradeorbit/bootleg#72]
```
crispin@vash:~/dev/epiccastle/bootleg$ bb --version
babashka v0.2.1
crispin@vash:~/dev/epiccastle/bootleg$ bootleg -v
Version: 0.1.9
crispin@vash:~/dev/epiccastle/bootleg$ cat test-bb-pod.clj
(ns static-website.core
  (:require [babashka.pods :as pods]))

(pods/load-pod "bootleg")
(require '[pod.retrogradeorbit.bootleg.utils :as utils])
$ bb test-bb-pod.clj
----- Error --------------------------------------------------------------------
Type:     java.lang.NullPointerException
Location: /home/crispin/dev/epiccastle/bootleg/test-bb-pod.clj:1:1

----- Context ------------------------------------------------------------------
1: (ns static-website.core
   ^--- 
2:   (:require [babashka.pods :as pods]))
3: 
4: (pods/load-pod "bootleg")
5: (require '[pod.retrogradeorbit.bootleg.utils :as utils])

----- Stack trace --------------------------------------------------------------
pod.retrogradeorbit.net.cgrand.xml - /home/crispin/dev/epiccastle/bootleg/test-bb-pod.clj:1:1
static-website.core                - /home/crispin/dev/epiccastle/bootleg/test-bb-pod.clj:4:1
```

**Comment by retrogradeorbit** [comment]
pod works with babashka v0.2.0 so use that for the time being.

**Comment by retrogradeorbit** [comment]
issue fixed in babashka v0.2.2. Upgrade babashka if you are having this issue.
